### ISSUE TITLE
Please add support for annotating the kafka/zookeper sets from configuration 

### ISSUE THREAD

**Issue by cameronbraid** [strimzi/strimzi-kafka-operator#847]
My use case is that I configure prometheus to scrape all pods using a path name based on an annotation like the following : 

```
apiVersion: v1
kind: Pod
metadata:
  annotations:
    prometheus.io/path: /metrics
```

Therefore it would be nice it the kafka/zookeeper resource definitions provided an annotations configuration option

**Comment by scholzj** [comment]
I think we currently support these annotations on services. So you can configure your Prometheus to scrape the services based on these annotations. There is also an existing issue #817 which should allow you to add custom annotations or labels to most of our resources. I think that should help in the future as well.

**Comment by scholzj** [comment]
This should be implemented in PR #1039 and will be release in 0.9.0. I will close this issue now. If you have any issues or problems, feel free to re-open it or open a new issue.
